crates/cxx-qt-lib/src/core/qdate.rs

```diff
@@ -48,6 +48,7 @@ mod ffi {
 
 /// The QDate class provides date functions.
 #[derive(Clone, PartialEq, Eq, PartialOrd, Ord)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 #[repr(C)]
 pub struct QDate {
```

Comment:
should we serialise as a number or the d,m,y dictionary?

Reply:
Yes gone with this. Although what format should be used? i've used `{"year":2023,"month":1,"day":1}` for now but is there a standard? like should it be y,m,d ? (and same for QTime, QTimeZone, QSize eg width or w etc).